examples/ref-templates/assets/htlc.py

```diff
@@ -41,16 +42,16 @@ def htlc(ARG_RCV,
     # Next, check that the Receiver field for this transaction is empty
     # Because this contract can approve transactions that close out its entire balance,
     # it should never have a receiver.
-    rec_field_check = Txn.receiver() == Global.zero_address()
+    rec_field_check = Txn.receiver() == ARG_RCV
 
     # Next, check that the Amount of algos transferred is 0. This is for the same reason as
     # above: we only allow transactions that close out this account completely, which
     # having a non-zero-address CloseRemainderTo will handle for us.
     amount_check = Txn.amount() == Int(0)
 
-    # Always verify that the RekeyTo property of any transaction is set to the ZeroAddress
+    # Always verify that the RekeyTo property of any transaction is set to the receiver addr
     # unless the contract is specifically involved ina rekeying operation.
-    rekey_check = Txn.rekey_to() == Global.zero_address()
+    rekey_check = Txn.rekey_to() == Txn.receiver()
```

Comment:
why do we change this? I think it should stay global zero address

Reply:
But as you said on discord 
*I think the receiver should be set to the same address as close reminder to. And in the smart contract we should check if receiver === the same address that was passed to the smart contract when deploying as close reminder to*

Reply:
corrected it